packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -268,6 +270,13 @@ class AppBar extends StatefulWidget implements PreferredSizeWidget {
   /// Defaults to being adapted to the current [TargetPlatform].
   final bool centerTitle;
 
+  /// The spacing around [title] content. This spacing is applied even if there
+  /// is no [leading] content or [actions]. If you want [title] to take all the
+  /// space available, set this value to 0.0.
+  ///
+  /// Defaults to [NavigationToolbar.kMiddleSpacing].
+  final double titleSpacing;
```

Comment:
This should say that the spacing is added on the left and right (or on the horizontal axis), so that people don't think it's added above or below.